smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/httpResponse/HttpResponseBindingErrorGeneratable.kt

```diff
@@ -1,15 +0,0 @@
-/*
- * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
- * SPDX-License-Identifier: Apache-2.0.
- */
-
-package software.amazon.smithy.swift.codegen.integration.httpResponse
-
-import software.amazon.smithy.codegen.core.Symbol
-import software.amazon.smithy.model.shapes.OperationShape
-import software.amazon.smithy.swift.codegen.integration.ProtocolGenerator
-
-interface HttpResponseBindingErrorGeneratable {
```

Comment:
Since there is now only one implementation of error generation, this protocol is unneeded